lib/entities/content-type.js

```diff
@@ -4,6 +4,7 @@
  */
 
 import cloneDeep from 'lodash/cloneDeep'
+import _ from 'lodash'
```

Comment:
we don't need to load the entire `lodash` here because it maybe lead to an increase in bundle size.
it's better to do it like htis `import findIndex from 'lodash/findIndex'` so we can benefit from the webpack tree shaking

Reply:
👍